### PR TITLE
LPD-21273 Quarantine failing Core Infrastructure tests

### PR DESCRIPTION
**Pull request by joshitagaki**
**Description**
Quarantine failing release tests to reduce impact on release testing analysis. Please unquarantine the tests as each fix is implemented in their respective follow up tickets.

**Ticket**
https://liferay.atlassian.net/browse/LPD-21273

### CODE REVIEW

**Comment by joshitagaki** [comment]
Hi team, please review these quarantined tests and feel free to forward if it looks good. Thank you!

**Comment by brunofalc** [comment]
cc @tinatian

**Comment by joshitagaki** [comment]
Hi @tinatian @brunofalc any updates on this review?

**Comment by tinatian** [comment]
@julschong Please take this over.

Thanks.
Tina

File: modules/util/portal-tools-db-upgrade-client/src/testFunctional/tests/DatabaseUpgradeClient.testcase [hunk 1]
```diff
@@ -96,6 +96,7 @@ definition {
 	@description = "LPS-158522: Upgrade client should not be able to run on versions greater than Java SE 8 "
 	@priority = 3
 	test CheckUpgradeClientJavaVersion {
+		property portal.upstream = "quarantine";
```
Comment (julschong) [inline modules/util/portal-tools-db-upgrade-client/src/testFunctional/tests/DatabaseUpgradeClient.testcase:99]: @joshitagaki This is under "Database Upgrade Tool" component, need to notify Database Infra Team that you are adding this test to quarantine. Their team seems to already has the test suite that will be able to catch quarantine.

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/customfields/cpcustomfields/CPCustomfields.testcase [hunk 2]
```diff
@@ -204,6 +204,8 @@ definition {
 	@description = "This is a use case for LPS-146795. The user admin could add new user with space character in geolocation custom field name."
 	@priority = 4
 	test AddNewUserWithGeolocationCustomFields {
+		property portal.upstream = "quarantine";
+
```
Comment (julschong) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/customfields/cpcustomfields/CPCustomfields.testcase:208]: @joshitagaki This is under "Instance Setting" component, need to notify Platform Experience Team that you are adding this test to quarantine. Their team will need to modify their test suite to catch quarantine

File: portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/portalconfiguration/systemmanagement/InstanceSettings.testcase [hunk 3]
```diff
@@ -1394,6 +1394,7 @@ definition {
 	@description = "This is a use case for LPS-156105. The test assert the value of mail setting for new instance is inherited from server level."
 	@priority = 2
 	test ViewInstanceMailSettingsValueIsInheritedFromServerLevel {
+		property portal.upstream = "quarantine";
```
Comment (julschong) [inline portal-web/test/functional/com/liferay/portalweb/tests/coreinfrastructure/administration/portalconfiguration/systemmanagement/InstanceSettings.testcase:1397]: @joshitagaki This is under "Instance Setting" component, need to notify Platform Experience Team that you are adding this test to quarantine. Their team will need to modify their test suite to catch quarantine

**Comment by julschong** [comment]
@joshitagaki Hi Josh,  
please see above comments and send PRs to the mentioned team so that they know what tests are added to their quarantined list. Once done, we can merge in.

File: modules/util/portal-tools-db-upgrade-client/src/testFunctional/tests/DatabaseUpgradeClient.testcase (hunk 1 above)
Comment (joshitagaki) [reply modules/util/portal-tools-db-upgrade-client/src/testFunctional/tests/DatabaseUpgradeClient.testcase]: @julschong just wanted to confirm was this the only test to be sent to another team?

File: modules/util/portal-tools-db-upgrade-client/src/testFunctional/tests/DatabaseUpgradeClient.testcase [hunk 4]
```diff
@@ -145,6 +146,7 @@ definition {
 	@description = "LPS-171012: Debug support is added for upgrade jvm"
 	@priority = 3
 	test CheckUpgradeDebugOptions {
+		property portal.upstream = "quarantine";
```
Comment (julschong) [inline modules/util/portal-tools-db-upgrade-client/src/testFunctional/tests/DatabaseUpgradeClient.testcase:148]: @joshitagaki Hi Josh, sorry this one too should be under Database Upgrade Tool

**Review by julschong — CHANGES_REQUESTED** [review]
@joshitagaki one more for Database Upgrade Tool team

**Comment by julschong** [comment]
ci:forward

**Comment by julschong** [comment]
manual see https://github.com/brianchandotcom/liferay-portal/pull/149646

**Pull request closed without merging**